docs/current/sdk/elixir/043817-install.md

```diff
@@ -0,0 +1,19 @@
+---
+slug: /043817/install
```

Comment:
I changing it to `/sdk/elixir/043817/install`. But it make netlify deployment failure https://app.netlify.com/sites/devel-docs-dagger-io/deploys/64d65d612227c90008a0bc48

Anyone know how to fix the issue?

Reply:
From netlify these are the broken links:

```
9:41:16 PM: [ERROR] Unable to build website for locale en.
9:41:16 PM: [ERROR] Error: Docusaurus found broken links!
9:41:16 PM: - On source page path = /sdk/elixir:
9:41:16 PM:    -> linking to /043817/install
9:41:16 PM:    -> linking to /756758/get-started
```

I have added some suggestions, can you please apply them and we can then debug further?

Reply:
I add `/sdk/elixir` prefix already. Let's see. :)

Reply:
@vikram-dagger deployed. I don't know why. 😂 

https://deploy-preview-5530--devel-docs-dagger-io.netlify.app/sdk/elixir/

Reply:
Checking the deploy, it seems to have built without error. The internal page links work as well. Can you clarify why you feel it is broken? Happy to help debug it further.

Reply:
I means the netlify build broken 4 days ago after I change the link. But today it’s work fine. 

BTW, the overall content looks ready now. :)